packages/vscode-autohotkey2-lsp/package.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: vscode-autohotkey2-lsp
+description: Autohotkey v2 Language Support using vscode-lsp.
+homepage: https://github.com/thqby/vscode-autohotkey2-lsp
+licenses:
+  - LGPL-3.0
+languages:
+  - AutoHotKey
+categories:
+  - LSP
+
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/thqby/vscode-autohotkey2-lsp@f358179042fd653a2c536ef24bb1b03898dfd5ca
+  build:
+    - target: win
+      run: |
+        cp ./tools/install.js .
+        rm * -Exclude install.js -Recurse -Force
+        node ./install.js
```

Comment:
I'm not a big fan of executing scripts like this for the following reasons:
- it may require additional system dependencies
- it may require a specific version of `node`
- there's no auditing or protection against the contents of it

It'd be much better if they were to publish it to `npm` instead. By being hosted on npm it's subject to removal and automatic auditing through services like https://socket.dev/.